translations/en.php

```diff
@@ -1,10 +1,15 @@
 <?php
 
 global $_MODULE;
-$_MODULE = [];
+$_MODULE = array();
```

Comment:
Convert this part manually again to [] (you can also use the fixer). otherwise, the validator won't accept it. It's not your fault, but PrestaShop fault, since it saves these kind of files using "array()"

```suggestion
$_MODULE = [];
```